index.html

```diff
@@ -44,5 +45,126 @@ <h3>LET’S CONNECT</h3>
       </ul>
     </div>
   </section>
+  <!--Works section-->
+
+  <section class="work-section">
+    <div class="card">
+      <img src="assets/images/tonic1.svg" alt="tonic1">
+      <div class="leftblock">
+      <div class="tonic">
+        <h3>Tonic</h3>
+        <ul>
+          <li><a href="#">Canopy <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button>See project</button>
+    </div>
+  </div>
+
+
+    <div class="card">
+      <img src="assets/images/poststories.svg" alt="tonic1">
+      <div class="tonic">
+        <h3>Multi-Post <br>
+          Stories</h3>
+        <ul>
+          <li><a class="Canopy" href="#">Canopy <img src="assets/images/Counter.svg" alt=""></i></a></li>
+          <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button>See project</button>
+    </div>
```

Comment:
As earlier requested, there are still a couple of concerns with the project card.
- the color of the tags are yet to be fixed
- the color of the button is not yet matching that on the figma.

Consider addressing all the above concerns in order to make your interface consistent with the figma and to give a potential recruiter the knowledge that you can replicate any kind of interface.


Your design
<img width="192" alt="oops" src="https://user-images.githubusercontent.com/84930355/236505821-3ae016ad-30a0-4f02-bc9a-79edc06c685d.PNG">

Figma
<img width="248" alt="hmmm" src="https://user-images.githubusercontent.com/84930355/236506254-ae49161b-beb5-4673-b952-00e435a8b9aa.PNG">